test/unit/data/events/get-events.test.js

```diff
@@ -0,0 +1,79 @@
+jest.mock('../../../../app/storage')
+const { getClient: mockGetClient, odata: mockOdata } = require('../../../../app/storage')
+
+const mockListEntities = jest.fn()
+
+const mockTableClient = {
+  listEntities: mockListEntities
+}
+
+const { PARTITION_KEY } = require('../../../mocks/values/partition-key')
+const { CATEGORY } = require('../../../mocks/values/category')
+const { PAYMENT_EVENT } = require('../../../../app/constants/event-types')
+
+const { stringifyEventData } = require('../../../helpers/stringify-event-data')
+
+let extractedEvent
+let enrichedEvent
+let events
+
+const { getEvents } = require('../../../../app/data/events/get-events')
```

Comment:
Should this be grouped with { stringifyEventData }?

Reply:
It's the focus so kept seperate

Reply:
If that's so, should it still be after the let's?